// Generated by pack tool 
// time: 09/02/15 16:58:34

package ProConfigSpace;

import "MonsterConfig.proto";
import "ItemBaseConfig.proto";
import "DropConfig.proto";
import "Monster_OtherConfig.proto";
import "ItemShowConfig.proto";
import "ItemSellConfig.proto";
message ProManagerConfig {
 optional string version = 1;
 repeated MonsterConfig monster = 2;
 repeated ItemBaseConfig itembase = 3;
 repeated DropConfig drop = 4;
 repeated Monster_OtherConfig monster_other = 5;
 repeated ItemShowConfig itemshow = 6;
 repeated ItemSellConfig itemsell = 7;
}
